dwave/system/temperatures.py

```diff
@@ -12,7 +12,7 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
-"""The following effective temperature and bias estimators are provided:
+r"""The following effective temperature and bias estimators are provided:
```

Comment:
Why the raw string here? I'm assuming this still doesn't fix the rendering of the math expressions below, but only instead shows the backslashes?

Reply:
The reason is 3.12 started raising `SyntaxWarning` for invalid escape sequences in strings. See [here](https://docs.python.org/3/whatsnew/3.12.html#other-language-changes). The solution is to use raw strings.

Reply:
Ah, that makes sense then. 👍